modules/drivers/lidar/proto/BUILD

```diff
@@ -51,4 +51,24 @@ proto_library(
     ],
 )
 
+proto_library(
+    name = "zvision_config_proto",
```

Comment:
do not add lidar proto here, move to own lidar dir, reference rslidar or livox lidar.